gui/src/main.rs

```diff
@@ -0,0 +1,35 @@
+mod game;
+mod game_over;
+mod tetrust;
+
+use bevy::prelude::*;
+
+use game::GamePlugin;
+use game_over::GameOverPlugin;
+
+#[derive(Debug, Default, States, Clone, Hash, PartialEq, Eq)]
+enum GameState {
+    GameOver,
+    #[default]
+    Game,
+}
+
+fn main() {
+    App::new()
+        .add_plugins((DefaultPlugins, GamePlugin, GameOverPlugin))
```

Comment:
FYI: 僕の環境では上手く動作しなかったため下記の修正をしてみましたが、環境に依存しそうなので、デフォルト設定を尊重したいと思います。

関連: https://github.com/bevyengine/bevy/issues/9975

```suggestion
        .add_plugins((
            DefaultPlugins.set(RenderPlugin {
                render_creation: RenderCreation::Automatic(WgpuSettings {
                    backends: Some(Backends::DX12),
                    ..default()
                }),
                ..default()
            }),
            GamePlugin,
            GameOverPlugin,
        ))
```